src/types/chart/bar_chart.ts

```diff
@@ -1,21 +1,6 @@
-import type { ChartConfiguration } from "chart.js";
-import { Color } from "../misc";
-import { LegendPosition, VerticalAxisPosition } from "./common_chart";
+import { ComboBarChartDefinition } from "./common_bar_combo";
 
-export interface BarChartDefinition {
+export interface BarChartDefinition extends ComboBarChartDefinition {
   readonly type: "bar";
-  readonly dataSets: string[];
-  readonly dataSetsHaveTitle: boolean;
-  readonly labelRange?: string;
-  readonly title: string;
-  readonly background?: Color;
-  readonly verticalAxisPosition: VerticalAxisPosition;
-  readonly legendPosition: LegendPosition;
   readonly stacked: boolean;
-  readonly aggregated?: boolean;
 }
-
-export type BarChartRuntime = {
```

Comment:
IMO it's better to keep `BarChartRuntime` and `ComboChartRuntime` even if it's the same type, I think it's clearer. WDYT ?

Reply:
I didn't have any preference, I just did it that way to avoid duplicating stuff, but you may be right :)